.github/CODEOWNERS

```diff
@@ -1,2 +1,6 @@
 # Every PR gets a review from an internal Expensify engineer
 * @Expensify/pullerbear
+
+# Assign the Design team to review changes to our styles & assets
+src/styles/ @Expensify/design @Expensify/pullerbear
+assets/ @Expensify/design @Expensify/pullerbear
```

Comment:
Not sure if it's necessary to add pullerbear here too since it's already an owner on `*` but haven't tested, so maybe it's necessary to make sure design doesn't override. NAB since it doesn't seem like a problem 🤷